.github/workflows/build.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
       - run: make gmp-bootstrap
       - run: make gtest-bootstrap
       - run: CFLAGS=-Werror make -j$(nproc) all plugin test
-      - run: meson builddir --werror
```

Comment:
Is this change intentional?

Reply:
Ah, I didn't realize this was intended for the openh264v2.3.1 branch and not master.

Reply:
just for test. no plan to merge